pkgs/development/python-modules/django_tables2/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, django }:
+
+buildPythonPackage rec {
+  pname = "django-tables2";
```

Comment:
directory and package name don't match

Reply:
underscores are generally used to denote specific versions.

Reply:
The package is called django-tables2, see https://pypi.org/project/django-tables2/